### PR TITLE
goofy little robots (arachne part... 3? 4?)

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

readds arachne in their original glory, resprited fully,

the old, infra, arachne borgs still exist. but i renamed them "slipper". why? uhh, idk. cause i thought the service one looked like a frog.

image

i think this is sprite freeze exempt? pretty sure. it doesn't touch markings and the like, so, it should be good. (:trollface:)

## How This Contributes To The Skyrat Roleplay Experience

uhhhhhhhhhhh. borgs are cool, and these are ""well-made."" and we have like 6 dogborgs for each dept. i think having some more "regular" ones is alright.

## Changelog

:cl: axietheaxolotl / viro
image: brand new arachne sprites!
code: old arachne renamed to slipper
/:cl:

### CODE REVIEW

**Comment by CandleJaxx** [comment]
finally. peace has returned

**Review by Ebin-Halcyon — APPROVED** [review]
Yeah, exempt. Doesn't touch anything that would cause a merge conflict.

**Comment by savoryrex** [comment]
yes. mis funks-a-BOT

**Review by desertmote — APPROVED** [review]
Great freaken job!

File: modular_skyrat/modules/altborgs/code/robot_model.dm [hunk 1]
```diff
@@ -66,7 +66,8 @@
 		"Butler" = list(SKIN_ICON_STATE = "service_m", SKIN_LIGHT_KEY = "service"),
 		"Meka" = list(SKIN_ICON_STATE = "mekaserve", SKIN_ICON = CYBORG_ICON_SERVICE_TALL, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_UNIQUETIP, R_TRAIT_TALL), SKIN_HAT_OFFSET = 15),
 		"Meka (Alt)" = list(SKIN_ICON_STATE = "mekaserve_alt", SKIN_LIGHT_KEY = "mekaserve", SKIN_ICON = CYBORG_ICON_SERVICE_TALL, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_UNIQUETIP, R_TRAIT_TALL), SKIN_HAT_OFFSET = 15),
-		"Arachne" = list(SKIN_ICON_STATE = "arachne_service", SKIN_ICON = CYBORG_ICON_SERVICE),
+		"ARACHNE" = list(SKIN_ICON_STATE = "arachne_service", SKIN_ICON = CYBORG_ICON_SERVICE),
```
Comment (jjpark-kb) [inline modular_skyrat/modules/altborgs/code/robot_model.dm:69]: why is this uppercase

File: modular_skyrat/modules/altborgs/code/robot_model.dm [hunk 2]
```diff
@@ -138,6 +140,7 @@
 		"Default" = list(SKIN_ICON_STATE = "engineer", SKIN_FEATURES = list(R_TRAIT_SMALL)),
 		"Meka" = list(SKIN_ICON_STATE = "mekaengi", SKIN_ICON = CYBORG_ICON_ENG_TALL, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_UNIQUETIP, R_TRAIT_TALL), SKIN_HAT_OFFSET = 15),
 		"Arachne" = list(SKIN_ICON_STATE = "arachne_engi", SKIN_ICON = CYBORG_ICON_ENG),
```
Comment (jjpark-kb) [inline modular_skyrat/modules/altborgs/code/robot_model.dm]: and this is lowercase

File: modular_skyrat/modules/altborgs/code/robot_model.dm (hunk 1 above)
Comment (axietheaxolotl) [reply modular_skyrat/modules/altborgs/code/robot_model.dm:69]: fixed

**Pull request merged**